frontend/src/modules/signup/view.jsx

```diff
@@ -168,7 +168,7 @@ const SignUp = ({ match: { params }, ...props }) => {
   const minHeightCard = innerHeight * 0.75;
 
   const storeData = UIStore.useState((s) => ({
-    stakeholders: s.stakeholders,
+    stakeholders: s.stakeholders.stakeholders,
```

Comment:
Trying to refresh the signup page and return this error result `TypeError: Cannot read property 'stakeholders' of undefined` on this line. I think it may be better to get the `.stakeholders` on the `stakeholder` API call.

Reply:
thanks @wayangalihpratama to catch this problem!
I adapted the code to avoid the null value
` stakeholders: s.stakeholders?.stakeholders,`
regarding the the FE api call, i left it as it was thus perhaps other pages/views will need other parts of the response (pagination, etc)... my take on that will be to delay decision/improvement on later/future requirements